Does not work with node v0.12.0

Full test log below

```
~/code/node-crawler$ npm install && npm test
 
> iconv@2.1.6 install /home/gerald/code/node-crawler/node_modules/iconv
> node-gyp rebuild

child_process: customFds option is deprecated, use stdio instead.
make: Entering directory `/home/gerald/code/node-crawler/node_modules/iconv/build'
  CC(target) Release/obj.target/libiconv/deps/libiconv/lib/iconv.o
  AR(target) Release/obj.target/iconv.a
  COPY Release/iconv.a
  CXX(target) Release/obj.target/iconv/src/binding.o
  SOLINK_MODULE(target) Release/obj.target/iconv.node
  SOLINK_MODULE(target) Release/obj.target/iconv.node: Finished
  COPY Release/iconv.node
make: Leaving directory `/home/gerald/code/node-crawler/node_modules/iconv/build'

> contextify@0.1.13 install /home/gerald/code/node-crawler/node_modules/jsdom/node_modules/contextify
> node-gyp rebuild

child_process: customFds option is deprecated, use stdio instead.
make: Entering directory `/home/gerald/code/node-crawler/node_modules/jsdom/node_modules/contextify/build'
  CXX(target) Release/obj.target/contextify/src/contextify.o
  SOLINK_MODULE(target) Release/obj.target/contextify.node
  SOLINK_MODULE(target) Release/obj.target/contextify.node: Finished
  COPY Release/contextify.node
make: Leaving directory `/home/gerald/code/node-crawler/node_modules/jsdom/node_modules/contextify/build'
mocha-testdata@1.1.0 node_modules/mocha-testdata

generic-pool@2.1.1 node_modules/generic-pool

iconv-lite@0.4.4 node_modules/iconv-lite

jschardet@1.1.0 node_modules/jschardet

lodash@2.4.1 node_modules/lodash

chai@1.9.2 node_modules/chai
├── assertion-error@1.0.0
└── deep-eql@0.1.3 (type-detect@0.1.1)

cheerio@0.18.0 node_modules/cheerio
├── entities@1.1.1
├── dom-serializer@0.0.1 (domelementtype@1.1.3)
├── CSSselect@0.4.1 (domutils@1.4.3, CSSwhat@0.4.7)
└── htmlparser2@3.8.2 (domelementtype@1.3.0, domutils@1.5.1, entities@1.0.0, domhandler@2.3.0, readable-stream@1.1.13)

sinon@1.11.1 node_modules/sinon
├── util@0.10.3 (inherits@2.0.1)
└── formatio@1.1.2 (samsam@1.1.2)

request@2.42.0 node_modules/request
├── caseless@0.6.0
├── json-stringify-safe@5.0.0
├── aws-sign2@0.5.0
├── forever-agent@0.5.2
├── stringstream@0.0.4
├── oauth-sign@0.4.0
├── tunnel-agent@0.4.0
├── qs@1.2.2
├── node-uuid@1.4.3
├── mime-types@1.0.2
├── bl@0.9.4 (readable-stream@1.0.33)
├── http-signature@0.10.1 (assert-plus@0.1.5, asn1@0.1.11, ctype@0.5.3)
├── tough-cookie@0.12.1 (punycode@1.3.2)
├── form-data@0.1.4 (mime@1.2.11, async@0.9.0, combined-stream@0.0.7)
└── hawk@1.1.1 (cryptiles@0.2.2, sntp@0.2.4, boom@0.4.2, hoek@0.9.1)

mocha@1.21.4 node_modules/mocha
├── diff@1.0.7
├── growl@1.8.1
├── commander@2.0.0
├── mkdirp@0.3.5
├── debug@2.1.3 (ms@0.7.0)
├── jade@0.26.3 (commander@0.6.1, mkdirp@0.3.0)
└── glob@3.2.3 (inherits@2.0.1, graceful-fs@2.0.3, minimatch@0.2.14)

iconv@2.1.6 node_modules/iconv
└── nan@1.6.2

jsdom@3.1.1 node_modules/jsdom
├── xmlhttprequest@1.7.0
├── xml-name-validator@1.0.0
├── browser-request@0.3.3
├── cssom@0.3.0
├── nwmatcher@1.3.4
├── parse5@1.4.1
├── htmlparser2@3.8.2 (domelementtype@1.3.0, entities@1.0.0, domhandler@2.3.0, readable-stream@1.1.13, domutils@1.5.1)
├── request@2.53.0 (caseless@0.9.0, json-stringify-safe@5.0.0, forever-agent@0.5.2, aws-sign2@0.5.0, stringstream@0.0.4, oauth-sign@0.6.0, tunnel-agent@0.4.0, isstream@0.1.2, node-uuid@1.4.3, qs@2.3.3, combined-stream@0.0.7, form-data@0.2.0, bl@0.9.4, http-signature@0.10.1, mime-types@2.0.10, tough-cookie@0.12.1, hawk@2.3.1)
├── cssstyle@0.2.23
└── contextify@0.1.13 (bindings@1.2.1, nan@1.5.3)

> crawler@0.4.2 test /home/gerald/code/node-crawler
> ./node_modules/mocha/bin/mocha --reporter spec --bail --timeout 10000 tests/*.js

child_process: customFds option is deprecated, use stdio instead.


  Cache features tests
    Cache
      - should crawl one url
    Skip Duplicate active
      1) should not skip one single url

  Encoding

  Errors
    timeout
    error status code

  Simple test

  Jquery testing
    Jquery parsing
    Jquery injection
    Cheerio specific test

  Links

  Limits
    rate limits

  Request tests

  Uri Options


  0 passing (10s)
  1 pending
  1 failing

  1) Cache features tests Skip Duplicate active should not skip one single url:
     Error: timeout of 10000ms exceeded
      at null.<anonymous> (/home/gerald/code/node-crawler/node_modules/mocha/lib/runnable.js:157:19)
      at Timer.listOnTimeout (timers.js:110:15)



npm ERR! Test failed.  See above for more details.
```

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
   "dependencies": {
     "cheerio": "0.18.0",
     "generic-pool": "2.1.1",
-    "iconv": "2.1.4",
+    "iconv": "2.1.6",
     "iconv-lite": "0.4.4",
     "jschardet": "1.1.0",
     "lodash": "2.4.1",

--- a/package.json
+++ b/package.json
@@ -51,7 +51,7 @@
   },
   "devDependencies": {
     "chai": "1.9.2",
-    "mocha": "1.21.4",
+    "mocha": "2.2.1",
     "mocha-testdata": "1.1.0",
     "sinon": "1.11.1",
     "jsdom": "3.1.1"

